force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -320,19 +411,17 @@ public with sharing class FlowControlLoopsCollections {
      * @param employeeName The name of the employee.
      * @return The salary of the employee, or null if the employee is not found in the Map.
      */
-    public static Integer getSalary(String employeeName) {
-        // Initialize a Map
 
-        // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+    public static Integer getSalary(String employeeName)
+    {
+        Map<String, Integer> employeeToSalary = new Map<String,Integer>(); // Initialize Map
 
-        // Get the salary of the employee
+        employeeToSalary.put('John Doe', 50000); // Populate Map
+        employeeToSalary.put('Jane Smith', 60000);
+        employeeToSalary.put('Sam Brown', 55000);
+        employeeToSalary.put('Alice Johnson', 65000);
 
-        // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        return employeeToSalary.get(employeeName); // Return employee salary
```

Comment:
According to the instructions, null should be returned if the employee is not found.